capris.html

```diff
@@ -6,8 +6,10 @@
   <title>Document</title>
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
   <link rel="stylesheet" href="main.css">
+  <script src="https://kit.fontawesome.com/3c70a135dc.js" crossorigin="anonymous"></script>
```

Comment:
Should this be at the bottom of the ```body``` tag?

Reply:
moved to bottom of body tag